src/lib/components/tablev2/SingleSelectableContent.tsx

```diff
@@ -154,7 +155,7 @@ export function SingleSelectableContent<
   }
   return (
     <>
-      <div className="thead" role="rowgroup">
+      <div className="thead" role="rowgroup" data-testid="table-head">
```

Comment:
Why is it necessary ?

Reply:
I'll check, I think it's not, probably just put it for consistency. Will remove it if it's not used of course